lib/modules/ens/index.js

```diff
@@ -5,10 +5,78 @@ const async = require('async');
 const embarkJsUtils = require('embarkjs').Utils;
 const reverseAddrSuffix = '.addr.reverse';
 
+const DEFAULT_ENS_CONTRACTS_CONFIG = {
+  "default": {
+    "contracts": {
+      "ENS": {
+        "deploy": false,
+        "silent": true
+      },
+      "ENSRegistry": {
+        "deploy": true,
+        "silent": true,
+        "args": []
+      },
+      "Resolver": {
+        "deploy": true,
+        "silent": true,
+        "args": ["$ENSRegistry"]
+      },
+      "FIFSRegistrar": {
+        "deploy": false
+      }
+    }
+  }
+};
+
+const ENS_CONTRACTS_CONFIG = {
+  "1": {
```

Comment:
Can we put those magic numbers into constants?

Reply:
If we don't want to introduce constants and use them as keys here, we can also introduce explicitly named config objects a ala `ROPSTEN_CONFIG`, `LIVENET_CONFIG` etc (or something along those lines...

Really my point is just that, prior to this change, it was much more obvious what every config is all about as it used to be named keys.

Reply:
Done, I have extracted the number into constants